login.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Blog</title>
+</head>
+<body>
+    <section>
+        <div class="welcome">
+            <h2 class="font">Welcome to</h2>
+            <h1 class="font" id="main-font">My Blog</h1>
+        </div> 
+        <div>
+            <form class="login-section">
+                <fieldset>
+                    <h2 class="login-text">Login</h2>
+                    <div class="login">
+                        <div>
+                            <label for="email">Email: </label>
+                            <input type="email" name="email" id="email" placeholder="E.g john123@gmail.com">
+                        </div>
+                        <div>
+                            <label for="password">Password: </label>
+                            <input type="password" name="password" id="password" placeholder="********">
+                        </div>
+                    </div>
+                    <div class="buttons">
+                        <a href="index.html" class="login-btn">

```

Comment:
Is the idea to have a link or a button for the signing/logging in?

Reply:
I haven't found a way of using JavaScript to make a button to open another page, but please inform me if you haave any idea